cmd/options.go

```diff
@@ -27,7 +27,7 @@ type options struct {
 
 func GatherInfluxOptions() db.InfluxDBOptions {
 	return db.InfluxDBOptions{
-		URL:                 environment.MustGetString(INFLUXDB_URL),
+		URL:                 environment.GetString(INFLUXDB_URL, ""),
```

Comment:
We changed that from `Must` to `GetString` because otherwise it would panic and won't log + close the program